### PR TITLE
Move date fields from send-money app and add 2-digit year entry

### PR DESCRIPTION
**Pull request by ushkarev**

### CODE REVIEW

**Comment by ibrechin** [comment]
That logic is going to stop working next century.

**Comment by ushkarev** [comment]
fair enough. how about now? lets ignore integer year overflow…

**Comment by ibrechin** [comment]
:+1:

File: mtp_common/assets/javascripts/modules/year-field.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// YearField module
+// Converts a 2-digit year to a full year
+/* globals exports, $ */
+'use strict';
+
+exports.YearField = {
+  selector: '.form-year-field',
```
Comment (maxf) [inline mtp_common/assets/javascripts/modules/year-field.js]: I'd suggest more of the functionality in the name, like YearFieldCompletion

**Pull request merged**